backend/README.md

```diff
@@ -10,8 +10,8 @@ On Debian Bullseye (11) with sudo:
 
 ```shell
 sudo apt update
-sudo apt install system_packages/debian/build_backend_packages.list
-sudo apt install system_packages/debian/run_backend_packages.list
+xargs sudo apt-get install < system_packages/debian/build_backend_packages.list
+xargs sudo apt install < system_packages/debian/run_backend_packages.list
```

Comment:
Why not purpose same syntax for two line ?

Reply:
Because I'm a dumb. haha, I fix it.